content/standards/openreferraluk/index.html.md

```diff
@@ -9,6 +9,7 @@ status: review
 dateAdded: 2020-12-16
 dateUpdated: 2021-07-05
 classification: Service, Product, Application, System or Platform Design
+licence_id: 'CC-BY-SA-4.0'
```

Comment:
Per a comment from Arnau, we should either be consistent (double quotes) or leave it off
```suggestion
licence_id: "CC-BY-SA-4.0"
```

Reply:
I've left it off all the others so removed the quotes from this one